packages/payment-integration-api/src/cart/buy-now-cart-request-body.ts

```diff
@@ -0,0 +1,18 @@
+import { CartSource } from './cart-source';
+
+interface LineItem {
+    productId: number;
+    quantity: number;
```

Comment:
Do we need to have variant ID as well here?

Reply:
As far as I know `variantId === optionId`

Reply:
For this case we have `optionSelections`

Reply:
There are various types of combinations as you can see here https://github.com/bigcommerce/bigcommerce/blob/master/app/Http/Storefront/Api/Cart/CartValidator.php#L88 it's an option.

Reply:
We don't need it here

Reply:
Maybe just add it as an optional now. Since partners might want to use if they can and storefront API allows for that.

Reply:
Sure